lib/translations.json

```diff
@@ -342,6 +342,14 @@
       "key": "marketing/signin/link",
       "value": "Jetzt anmelden"
     },
+    {
+      "key": "marketing/claim",
+      "value": "Sie haben einen Gutschein? {claimLink}"
+    },
+    {
+      "key": "marketing/claim/link",
+      "value": "Gutschein einlösen"
```

Comment:
Eigentlich ein sehr guter Linktext, aber mit der Frage vorher klingts etwas redundant/holprig/technisch:
`Sie haben einen Gutschein? Gutschein einlösen`

Alternative 1:
`Sie haben einen Gutschein? Jetzt einlösen`
Vorteil: Liest sich besser.
Nachteil: Der Linktext verliert an Semantik.

Alternative 2:
Nur `Gutschein einlösen`
Vorteil: Kurz und semantisch.
Nachteil: Zumindest in einem von den beiden Kontexten evtl. *zu* kurz.

Wollts nur mal aufwerfen. Nicht launch-blocking.